docs/v7_migration.rst

```diff
@@ -224,6 +208,42 @@ errors:
 - ``TransactionIndexingInProgress``
 
 
+End of Support and Feature Removals
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Python 3.7 Support Dropped
+``````````````````````````
+
+Python 3.7 support has been dropped in favor of Python 3.8+. Python 3.7 is no longer
+supported by the Python core team, and we want to focus our efforts on supporting
+the latest versions of Python.
+
+
+EthPM Module Removed
+````````````````````
+
+The EthPM module has been removed from the library. It was not widely used and has not
+been functional since around October 2022. It was deprecated in ``v6`` and has been
```

Comment:
just a passing curiosity - it hasn't been functional? or the tests haven't been functional?

Reply:
Not sure tbh. I just moved this section over. This was reviewed and merged somewhere else.

Reply:
Using it relied on a free IPFS portal provided by Infura. Once Infura started requiring an API key to access it (in October '22), EthPM no longer worked. I looked at updating it to allow the user to provide their own key, but we determined that it wasn't getting enough use to make the updates worthwhile.

Reply:
got it, thanks! 🚢

Reply:
> Using it relied on a free IPFS portal provided by Infura. Once Infura started requiring an API key to access it (in October '22), EthPM no longer worked. I looked at updating it to allow the user to provide their own key, but we determined that it wasn't getting enough use to make the updates worthwhile.

Ah yeah, now I remember that context 👌🏼